aws_organizations/release.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Usage: ./release.sh <S3_Bucket>
```

Comment:
Is there a reason we want/need a separate release script for this, rather than just doing it in the main release script?

Reply:
For testing this is more convenient. I modified it slightly because the [original](https://github.com/DataDog/cloudformation-template/blob/master/aws_quickstart/release.sh#L51) creates the quick-create url, whereas this one is just hard-coding the stack set create url.

Reply:
Here's an example:

```
Done uploading the template. Navigate to https://us-east-1.console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacksets/create and use the template URL below.
https://datadog-cloudformation-template-test.s3.amazonaws.com/aws/main_organizations.yaml
Done!
```